modules/assignment/src/main/java/com/intuit/wasabi/assignment/impl/AssignmentsImpl.java

```diff
@@ -503,9 +550,21 @@ public Assignment getAssignment(User.ID userID, Application.Name applicationName
                 experimentBatch.getLabels().remove(experiment.getLabel());
             }
         }
+        if(LOGGER.isDebugEnabled()) LOGGER.debug("allAssignments: {} ", allAssignments);
```

Comment:
isDebugEnabled() mostly clustters the code that does very little if anything at all since the logger.debug will evaluate if the statement will need to be logged based on the log level.  Unless you doing things like computation on the log statement it is usually not worth it to add the isDebugEnabled statement